syntax = "proto3";
package dfi;

message DFIRetrieveFlowHandleRequest {
	string name = 1;
}
